radboudumc-template/templates/publications.html

```diff
@@ -5,40 +5,50 @@
 {% block content %}
     {% include 'blocks/breadcrumbs.html' %}
     <div class="container">
-    <h1>{{ page.title }}</h1>
 
 
+    <h1>{{ page.title  }} </h1>
 
-    {% set list_years = [] %}
-    {% set list_pub_type = [] %}
+    {% set list_pub_type = ['article', 'preprint', 'inproceedings', 'conference', 'phdthesis', 'mastersthesis', 'book', 'other'] %}
+    {% set years = [] %}    
+    {% set bib_keys = [] %}
+    {% for person in pages|reverse %}
+	{% if person.parent.url == 'members/' %}
+	   {% set person_key = 'ja' %}  
+	   {% if person.pub_name %}
+		{% set person_key = person.pub_name.replace(' ', '-') %}  
+	   {% else %}
+		{% set person_key = person.name.replace(' ', '-') %}  
+	   {% endif %}
+	   {% if person_key in author_keys %}
+	   	{% for bibkey in author_keys[person_key]  %}
+			{% if bibkey not in bib_keys %}
+			    {% set _ = bib_keys.append(bibkey) %}
+			{% endif %}
+		{% endfor %}
+	   {% endif %}
+	{% endif %}
+    {% endfor %}
 
 
-    {% if bibkeys_html %}
-    {% for bibkey in bibkeys_html.keys() %}
-        {# Getting list of years #}
-        {% if bibkeys_html[bibkey]['year'] not in list_years %}
-            {% if list_years.append(bibkeys_html[bibkey]['year']) %}{% endif %}
-        {% endif %}
-        {# Getting list of pub_type #}
-        {% if bibkeys_html[bibkey]['pub_type'] not in list_pub_type %}
-            {% if list_pub_type.append(bibkeys_html[bibkey]['pub_type']) %}{% endif %}
-        {% endif %}
+    {% for bibkey in bib_keys if bib_items[bibkey]['year'] not in years %}
+      {% set _ = years.append(bib_items[bibkey]['year']) %}
     {% endfor %}
-    {% endif %}
-    <style>
-      input[name="year-selection"]:checked + label {
-      font-weight: bold;
-      }
-      input[name="year-selection"]:hover + label {
-      font-weight: bold;
-      text-decoration: none;
-      }
 
+    <style>
+	input[name="year-selection"]:checked + label {
+	    font-weight: bold;
+	}
+	input[name="year-selection"]:hover + label {
+	    font-weight: bold;
+	    text-decoration: none;
+	}
```

Comment:
Please move this to the css file. Also, the buttons are now missing a cursor: pointer tag and jump when hovered.